pom.xml

```diff
@@ -26,6 +26,7 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
+
```

Comment:
Revert this change, when you don't have change don't add any space and bring it to pull request - This is bad practice